graylog2-rest-models/src/main/java/org/graylog2/rest/models/system/indexer/responses/IndexRangeSummary.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * This file is part of Graylog.
+ *
+ * Graylog is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Graylog is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Graylog.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.graylog2.rest.models.system.indexer.responses;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.auto.value.AutoValue;
+import org.joda.time.DateTime;
+
+import javax.annotation.Nullable;
+
+@AutoValue
+@JsonAutoDetect
+public abstract class IndexRangeSummary {
+    @JsonProperty("index_name")
+    public abstract String indexName();
+
+    @Nullable @JsonProperty("calculated_at")
+    public abstract DateTime calculatedAt();
+
+    @JsonProperty("start")
+    public abstract DateTime start();
+
+    @JsonProperty("calculation_took_ms")
+    public abstract int calculationTookMs();
+
+    @JsonCreator
+    public static IndexRangeSummary create(@JsonProperty("index_name") String indexName,
+                                           @Nullable @JsonProperty("calculated_at") DateTime calculatedAt,
+                                           @JsonProperty("start") DateTime start,
+                                           @Nullable @JsonProperty("calculation_took_ms") int calculationTookMs) {
```

Comment:
A primitive value can never be `null`, so the `@Nullable` annotation is obsolete.